fairroot/base/CMakeLists.txt

```
@@ -80,6 +78,13 @@ if(BUILD_PROOF_SUPPORT)
   )
 endif()
 
+if(BUILD_EVENT_BUILDER)
+  list(APPEND sources
+    event/FairEventBuilder.cxx
+    event/FairEventBuilderManager.cxx
+  )
+endif()
```

You need to pass on a define, so that the linkdef works.

```suggestion
  )
  target_compile_definitions(${target} PRIVATE BUILD_EVENT_BUILDER)
endif()
```

---

Added.